target/linux/ramips/image/lzma-loader/src/loader.c

```diff
@@ -28,8 +28,7 @@
 #include "printf.h"
 #include "LzmaDecode.h"
 
-#define AR71XX_FLASH_START	0x1f000000
-#define AR71XX_FLASH_END	0x1fe00000
+#define FLASH_START	0x1c000000
```

Comment:
Please indent `0x1c000000` to the other values!
```suggestion
#define FLASH_START		0x1c000000
```

Reply:
Never mind! See 981213's [comment](https://github.com/openwrt/openwrt/pull/2645#discussion_r480949301)